benchmarks/benchmarks/stats.py

```diff
@@ -41,6 +42,26 @@ def time_barnard_exact(self, alternative):
         resBarnard = stats.barnard_exact(self.a, alternative=alternative)
 
 
+class Kendalltau(Benchmark):
+    param_names = ['nan_policy','method','variant']
+    params = [
+        ['propagate', 'raise', 'omit'],
+        ['auto', 'asymptotic', 'exact'],
+        ['b', 'c']
+    ]
+
+    def setup(self, nan_policy, method, variant):
+        a = [x for x in range(0, 200)]
+        random.shuffle(a)
+        b = [x for x in range(0, 200)]
+        random.shuffle(b)
+        self.a = a
+        self.b = b
+
+    def time_kendalltau(self, nan_policy, method, variant):
+        tau, p_value = stats.kendalltau(self.a, self.b, nan_policy = nan_policy, method = method, variant = variant)
```

Comment:
minor: PEP8 convention to remove the spaces around `=` for the kwargs

Reply:
Sorry, I didn't notice that before ... I just made 2 commits to fix it, could you help me review it again? Thanks !

Also, I noticed that I've made 3 commits for this small benchmark, is this too many? I can checkout a new branch from master and submit a new PR with only one commit...

Reply:
We can squash merge.

Reply:
Thanks for your suggestion, that's helpful! I don't know squash merge before and now I think it is good choice!